src/main/java/bio/terra/cloudfunctions/streaming/GcsBQ.java

```diff
@@ -20,6 +37,87 @@
 
 public class GcsBQ implements BackgroundFunction<GCSEvent> {
   private static final Logger logger = Logger.getLogger(GcsBQ.class.getName());
+  private static final String DATASET = "simple_stream_dataset";
+  private static final String TABLE = "SUMMARY_testRun";
```

Comment:
In the script that uploads to BQ in the test runner library, there are 3 [tables](https://github.com/DataBiosphere/terra-test-runner/blob/f963972154a550796a0102d0b85c1ebe5695b4d3/src/main/java/scripts/uploadscripts/SummariesToBigQuery.java#L62): `testRun`, `testScriptResults`, `measurementCollection`. I don't think you need the `measurementCollection` one here, but curious why you also dropped the `testScriptResults` one?

Reply:
This PR focussed on making sure that streaming works for one of the tables - `SUMMARY_testRun` table was chosen for this work. The remaining tables (`testScriptResults`, `ENV_componentVersion`) will be handled in a separate ticket.